lib/pf/freeradius.pm

```diff
@@ -155,9 +154,57 @@ sub freeradius_populate_nas_config {
         _insert_nas_bulk( @rows );
     }
     _delete_expired($timestamp);
+    validate_radius_nas_table($timestamp);
+}
+
+our $validate_radius_nas_table_sql = <<SQL;
+    SELECT
+        (SELECT COUNT(1) FROM radius_nas) = (SELECT COUNT(1) FROM radius_nas WHERE config_timestamp = ?) AS config_valid,
+        (SELECT COUNT(1) FROM radius_nas WHERE config_timestamp > ?) as count_greater_than,
```

Comment:
This counter and the one below don't seem to be used. Any reason for keeping them?